airflow/utils/context.pyi

```diff
@@ -48,7 +48,7 @@ class VariableAccessor:
 class ConnectionAccessor:
     def get(self, key: str, default_conn: Any = None) -> Any: ...
 
-class Context(TypedDict, total=False):
+class Context(TypedDict, total=False):  # type: ignore
```

Comment:
What is the error you get if you don’t ignore here? (For debugging reference later.)

Reply:
Here:
```
airflow/utils/context.pyi:51: error: Unexpected keyword argument "total" for "__init_subclass__"
of "object"
    class Context(TypedDict, total=False):  
    ^
Found 1 error in 1 file (checked 60 source files)
```

Reply:
`total=False` is Python 3.8 + feature https://docs.python.org/3/whatsnew/3.8.html
Not sure if  it will be really helpful. I'd say we should specify all possible valuesi in Context.

Reply:
Have removed it since we still support python 3.7